docs/api.md

```diff
@@ -73,3 +73,80 @@ function example_civicrm_mosaicoConfig(&$config) {
   $config['tinymceConfigFull']['toolbar1'] = 'bold italic removeformat | link unlink | civicrmtoken | pastetext code';
 }
 ```
+
+## hook_civicrm_mosaicoStyleUrlsAlter(&$styleUrls)
+
+`$styleUrls` is an array of URLs for stylesheets to be loaded in the Mosaico composer interface. Alter (add, remove, change) array elements as needed.
+
+Example from  [Campaign Advocacy](https://github.com/twomice/civicrm-campaignadvocacy):
+
+```php
+function campaignadv_civicrm_mosaicoStyleUrlsAlter(&$styleUrls) {
+  $res = CRM_Core_Resources::singleton();
+  $snippets = array_merge(Civi::service('bundle.coreResources')->getAll(), Civi::service('bundle.coreStyles')->getAll());
```

Comment:
@lucky091588 I recommend removing lines 86-109 because they'll be confusing for people who just want to know how this hook works. I would say that lines 110-112 are enough for this example.